boxoffice/assets/js/models/util.js

```diff
@@ -43,6 +54,7 @@ export const post = function (config) {
     url: config.url,
     type: 'POST',
     data: config.data,
+    processData: config.processData ? config.processData : false,
```

Comment:
Can you add a comment to describe what this means?

Reply:
Fixed